src/main/groovy/com/twcable/grabbit/client/servlets/GrabbitJobServlet.groovy

```diff
@@ -49,7 +49,7 @@ import static javax.servlet.http.HttpServletResponse.SC_OK
  */
 @Slf4j
 @CompileStatic
-@SlingServlet(methods = ['GET', 'PUT'], resourceTypes = ['twcable:grabbit/job'])
+@SlingServlet(methods = ['GET', 'PUT', 'POST'], resourceTypes = ['twcable:grabbit/job'])
```

Comment:
Yeah, I see your point.

 I initially suggested the DELETE verb for removing jobs from the repository. If you remove a job from the repository, ideally you would make sure it is stopped as well (which we aren't doing currently in the clean servlet I don't think, but..); and possibly vice-versa. So, maybe we can consolidate stopping a job, and deleting a job in the future, because really having them be seperate concepts is sort of an artificial creation..

So I guess if I had to lean one way, it might be DELETE